exporter/signalfxexporter/README.md

```diff
@@ -1,14 +1,21 @@
 # SignalFx Metrics Exporter
 
+| Status                   |                       |
+| ------------------------ |-----------------------|
+| Stability                | [beta]                |
+| Supported pipeline types | traces, logs, metrics |
+| Distributions            | [contrib]             |
+
+[beta]:https://github.com/open-telemetry/opentelemetry-collector#beta
+[contrib]:https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
+
 This exporter can be used to send metrics, events, and trace correlation to SignalFx.
 
 Apart from metrics, the exporter is also capable of sending metric metadata
 (properties and tags) to SignalFx. Currently, only metric metadata updates from
 the [k8s_cluster receiver](../../receiver/k8sclusterreceiver/README.md) are
 supported.
 
-Supported pipeline types: logs (events), metrics, traces (trace to metric correlation only)
```

Comment:
Can you please move the whole sting `logs (events), metrics, traces (trace to metric correlation only)` to the table. This is pretty important information.

Reply:
done